Section 9 - Example 
Calculation of a model with 10 gr per kvdm wing load. 
The following values are selected or are given by the class rules: cm.

* Total span - 49 cm.
* Body length - 45 cm 
* Hook distance - 40 cm
* Body section - 2.3 cm
* Side ratio - 10 cm

* Effective span becomes L in cm. is 95% of the total wingspan 
* For an aspect ratio of 10 the average wing width B in cm. is L/10 
* The wing load bearing area F in kvdm is L times B 
* The minimum weigh Q in grams is 10 timesthe total wing area
* It is intended to use a gear with specifed ratio u 
* It is intended for a single rubber loop, so a = 1

In [43]:
import numpy as np

The parameters appear in the following code cell

In [44]:
L = 47
B = 4.7
F = L*B/100
Q = 22
u = 3.4
a = 1

The motor cross section according the following equation

$G = 0.08 \sqrt [3] {(\frac {u} {a} Q L)^2} $

In [45]:
G = 0.08*(((u/a)*Q*L)**2)**0.333
print("G = ", '{:0.3f}'.format(G), "kvdm")

G =  18.396 kvdm


The closest value which can be obtained with 2.4 or 4 kvdm rubber strip is 6 stands of 2.7 kvdm => 16.2 kvdm or 4 strands of 4 kvdm => 16 kvdm.
We choose G = 16 kvdm.  

The maximum motor turns, V, that can be wound, with stretching

$V = 65 \frac {H} {\sqrt {G}} $

In [46]:
H = 40
G = 16
V = 65*H/G**(0.5)
print ("V = ", '{:0.0f}'.format(V), "turns") 

V =  650 turns


Thus the propeller will be able to have u times V or 3.4 x 650 = 2200 revolutions.

The maximum torque of the motor, Mmax, in mmgr will be

$Mmax = 30 \frac {a} {u} \sqrt { g^3}$

In [47]:
Mmax = 30*(a/u)*(G**3)**(0.5)
print ("Mmax =",  '{:0.1f}'.format(Mmax), "mmgr")

Mmax = 564.7 mmgr


A suitable propeller pitch, S, in cm, is

$S = 0.08 \frac {Mmax} {F} $

In [48]:
S = 0.08*Mmax/F
print ("S =",  '{:0.1f}'.format(S), "cm")

S = 20.5 cm


The propeller diameter, D, in cm  should then be

$D = 3 \sqrt {S} \sqrt [4] {F} $

In [49]:
D = 3*S**0.5*F**0.25
print ("D =",  '{:0.1f}'.format(D), "cm")

D = 16.5 cm


The largest blade width is chosen to be  D / 8 or 2 cm 

The rubber weight K in grams is

$K = 0.009 a H G $

In [50]:
K = 0.009*a*H*G
print ("K =",  '{:0.2f}'.format(K), "grams")

K = 5.76 grams


The total flight time in seconds is

$T = 580 \frac {K} {Q} \sqrt { \frac {F} {Q} } \sqrt [3] { \frac {L} {B} }$


In [51]:
T = 580*(K/Q)*(F/Q)**0.5*(L/B)**0.333
print ("T =",  '{:0.1f}'.format(T), "seconds")

T = 103.6 seconds


The average torque of the motor, Mmax, in mmgr will be

$Mmed = 12 \frac {a} {u} \sqrt { g^3}$

In [52]:
Mmed = 12*(a/u)*(G**3)**(0.5)
print ("Mmed =",  '{:0.2f}'.format(Mmed), "mmgr")

Mmed = 225.88 mmgr


The average propeller speed in revolutins per minute will be

$n = 160000 \frac {\sqrt {Mmed}} {D^2 \sqrt{S}}$

In [53]:
n = 160000*Mmed**0.5/((D**2)*(S**0.5))
print ("n =",  '{:0.0f}'.format(n), "RPM")

n = 1944 RPM


The time for the motor to unwind is

$T_m = 60 \frac {V*u} {n}$

In [54]:
Tm = 60*V*u/n
print ("Tm =",  '{:0.1f}'.format(Tm), "seconds")

Tm = 68.2 seconds


The flight speed in meter per second is

$v_g = 1.9 \sqrt  { \frac {S}{F}}$

In [55]:
vg = 1.9*(S/F)**0.5
print ("vg =",  '{:0.1f}'.format(vg), "m/s ")

vg = 5.8 m/s 


The glide ratio is

$g = 0.2 \sqrt [3] { \frac {B} {l} } $

In [56]:
g = 0.2*(B/L)**0.333
print ("g =",  '{:0.3f}'.format(g), " ")

g = 0.093  


The sinking speed is 

$v_s = 0.38 \sqrt {S}{F} \sqrt [3] \frac {B} {L} $

In [57]:
vs = 0.38*(S/F)**0.5*(B/L)**0.333
print ("vs =",  '{:0.2f}'.format(vs), "m/s")

vs = 0.54 m/s


The greatest height above ground is

$Y = 0.56*(T - Tm) $

In [58]:
Y = 0.56*(T - Tm)
print ("Y =",  '{:0.1f}'.format(Y), "m")

Y = 19.8 m


The straight line flight distance will be

$X = vg * T $

In [59]:
X = vg *T
print ("x =",  '{:0.1f}'.format(X), "m")

x = 598.9 m


The model can be built so that without rubbber it weight 15 grams, which should be possible with the use of balsa for the propeller and with japanese paper for the covering material.

The power deivered by the motor in horsepower is

$Pwr_m = \frac {2 \pi M_med n_med} {1000 . 1000 . 60 . 75} $

In [60]:
Pwrm = (2*3.14159*Mmed*n)/(1000*1000*60*75)

print ("Pwrm =",  '{:0.6f}'.format(Pm), "hp")

Pwrm = 0.000613 hp


The proepller thrust in grams with a slip, s, of 15% is

$P = 0.06 \sqrt {s + 15} \frac {Mmed} {S} $

In [61]:
s = 15
P = 0.06*(s+15)**0.5*Mmed/S
print ("P =",  '{:0.2f}'.format(P), "gram")

P = 3.63 gram


The power delivered by the propeller is

$Pwrp = \frac {P v_g} {1000 * 75} $

In [62]:
Pwrp = (P*vg)/(1000*75)

print ("Pwrp =",  '{:0.6f}'.format(Pwrp), "m")

Pwrp = 0.000280 m


The propeller effciency is

$\eta = Pwrp/Pwrm $

In [64]:
efficiency = (Pwrp/Pwrm)*100
print ("efficiency =",  '{:0.1f}'.format(efficiency), "%")

efficiency = 45.6 %
